openshift/02-apicast-builder-imagestream.yml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: ImageStream
+metadata:
+  name: apicast
+  labels:
+    app: apicast
+spec:
+  tags:
+  - name: master-builder
+    from:
+      kind: DockerImage
+      name: quay.io/3scale/apicast:master-builder
```

Comment:
Would be good to have a comment pointing to how is this refreshed.
https://github.com/3scale/apicast-cloud-hosted/pull/4#discussion_r169143362

Reply:
TBH, I don't know how `quay.io/3scale/apicast:master-builder` is refreshed either...  Or are you talking about the scheduled imports from the registry?

Reply:
Yep. Scheduled imports. 

Those quay images are built when master branch is updated.

Reply:
Done in 287e33c18f82a14d84e86f17a83dc1c5239aabcd